trustfall_core/src/graphql_query/directives.rs

```diff
@@ -1,3 +1,5 @@
+//! Directives in GraphQL can be identified by staring with `@`. While
+//! `trustfall_core` doesn't support all GraphQL directives, some are available.
```

Comment:
Yes, `graphql_query` definitely was a source of confusion for me, since it imples that GraphQL is a core part of Trustfall (it is in `trustfall_core/src` after all!). But thanks for suggestions!